### DISCUSSION TITLE
[solved] tox discord server chimera

### DISCUSSION THREAD

**Discussion by ssbarnea** [tox-dev/tox#1879] [Ideas]
Trying to identify the IM platform where to engage with other tox users seams to be futile attempt.

As @gaborbernat indicated on old Gitter tox-dev channel about gitter deprecation, Discord was mentioned with a link that did not work.

I tried to Google, and apparently Google reports that main page may contain a reference to Discord, but when you try to search for it of the page, nothing is to be found, likely the indexed version had it but was removed since.

![](https://sbarnea.com/ss/Screen-Shot-2021-01-27-13-37-08.25.png)

Is that channel so secret? ;)

**Comment by gaborbernat** [comment]
https://discord.gg/edtj86wzBX had posted there an extra )

**Reply by ssbarnea** [reply]
I should have guessed that adding an "extra" is where everything starts to fall apart (reference to packaging).
